src/raftstore/store/peer.rs

```diff
@@ -476,10 +484,11 @@ impl Peer {
             // for read-only, if we don't care stale read, we can
             // execute these commands immediately in leader.
```

Comment:
// For the read-only commands, if we don't care about stale read, we can
              // execute these commands directly in the leader.